<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDetailDlg</class>
 <widget class="QWidget" name="LogDetailDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogDetailDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame4">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"> QWidget#frame4
 {
	border:0px solid #cccccc;
	border-bottom-width:1px;
	border-left-width:1px;
	border-right-width:1px;
	background-color:white;
 }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_22">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="chkOnlineLog">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QRadioButton:hover
{
	background-color:rgba(100,100,100, 50);
}

QRadioButton:checked
{
	background-color:#cccccc;
}

 QRadioButton::indicator 
{
     width: 10px;
     height: 32px;
	 image:none;
 }</string>
        </property>
        <property name="text">
         <string>上下线日志</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkLogDownload">
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QRadioButton:hover
{
	background-color:rgba(100,100,100, 50);
}

QRadioButton:checked
{
	background-color:#cccccc;
}

 QRadioButton::indicator 
{
     width: 10px;
     height: 32px;
	 image:none;
 }</string>
        </property>
        <property name="text">
         <string>文件下载记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkLogUpload">
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QRadioButton:hover
{
	background-color:rgba(100,100,100, 50);
}

QRadioButton:checked
{
	background-color:#cccccc;
}

 QRadioButton::indicator 
{
     width: 10px;
     height: 32px;
	 image:none;
 }</string>
        </property>
        <property name="text">
         <string>文件上传记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkLogPlugin">
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QRadioButton:hover
{
	background-color:rgba(100,100,100, 50);
}

QRadioButton:checked
{
	background-color:#cccccc;
}

 QRadioButton::indicator 
{
     width: 10px;
     height: 32px;
	 image:none;
 }</string>
        </property>
        <property name="text">
         <string>插件数据回传记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkDetailLog">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QRadioButton:hover
{
	background-color:rgba(100,100,100, 50);
}

QRadioButton:checked
{
	background-color:#cccccc;
}

 QRadioButton::indicator 
{
     width: 10px;
     height: 32px;
	 image:none;
 }</string>
        </property>
        <property name="text">
         <string>全部日志</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stkLog">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_16">
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PageTableWidget" name="tbLog" native="true">
         <property name="maximumSize">
          <size>
           <width>1677215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_18">
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PageTableWidget" name="tbDetailLog" native="true">
         <property name="maximumSize">
          <size>
           <width>1677215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_19">
      <layout class="QHBoxLayout" name="horizontalLayout_27">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PageTableWidget" name="tbLogDownload" native="true">
         <property name="maximumSize">
          <size>
           <width>1677215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_26">
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PageTableWidget" name="tbLogUpload" native="true">
         <property name="maximumSize">
          <size>
           <width>1677215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_27">
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PageTableWidget" name="tbLogPlugin" native="true">
         <property name="maximumSize">
          <size>
           <width>1677215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PageTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">pagetablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
